game-rules.tex

```diff
@@ -3,51 +3,27 @@ \section{Game Rules}
-  \item The arena contains 24 tokens.
-  \item There are nine scoring zones in the arena, arranged in a $3\times3$
-        grid. The central scoring zone is raised.
-  \item A scoring zone is considered to be `controlled' by a team if and only if:
-    \begin{enumerate}
-      \item the team has tokens in the zone.
-      \item no other team has a greater `weighting of tokens' in the zone.
-    \end{enumerate}
-  \item The `weighting of tokens' in a zone for a team is calculated from the sum of token weights in the zone.
-      Token weights are determined as follows:
-      \begin{enumerate}
-            \item A token `fully in' the zone has a weighting of $2$.
-            \item A token `partially in' the zone has a weighting of $1$.
-    \end{enumerate}
-  \item A `controlled region' is defined as follows:
-    \begin{enumerate}
-      \item A `controlled region' is a group of directly adjacent scoring zones controlled by the same team.
-      \item The value of a `controlled region' is calculated by summing the value of all zones within the `controlled region'.
-  \item All scoring zones have a value of exactly $1$, with the exception of the central raised zone which has a value of exactly $2$.
-    \end{enumerate}
-  \item At the end of a match, robots earn points as follows:
-    \begin{enumerate}
-      \item $1$ point for moving entirely out of the starting area.
-      \item $n^2$ points for each `controlled region' where $n$ is the value of the `controlled region'. 
-    \end{enumerate}
-  \item A token is considered to be `fully in' a zone if the vertical projection of token
-        is entirely within the scoring zone.
-  \item A token is considered to be `partially in' a zone if the vertical projection of token
-        is partially within the scoring zone.
-  \item During a match, a robot may move or interact with any token in the arena.
-  \item Participating teams must present their robots to match officials before
-        the start of matches, as regulated by the match officials. Non-compliant
-        teams may be disallowed from participating.
-  \item There will be up to 4 robots in each match.
-  \item \org may have any number of match officials within the arena, including
-        during the course of matches.
-  \item At the start of each match, robots must be entirely within their
-        starting areas.
-%   \item At the start of each match, teams will be permitted to lean into the
-%         arena and start their robots.
-  \item Each match lasts $120$ seconds.
+        Specification~\ref{spec:arena}. The objective is to collect containers
+        and deposit them in scoring zones.
+  \item The game is played by two teams, each team has two robots: a crane and
+        a forklift.
+  \item The forklift is used to pick up loose containers and move them to the
+        docking area. Each team's forklift starts in their starting area.
+  \item The arena contains 16 containers. The containers are labeled with
```

Comment:
With https://github.com/sourcebots/competition-simulator/pull/67, 2 tokens now start in the docking area, so there's something for the cranes to do at the start.

This also means there are now 18 tokens.